testplan/web_ui/testing/src/Toolbar/InteractiveButtons.js

```diff
@@ -46,4 +65,204 @@ const ResetButton = (props) => {
   }
 };
 
-export {ResetButton};
+
+const showHistory = (historyExporters) => {
```

Comment:
more of a getHistoryTable than showHistory as that sounds like you return a flag if show or not show